formats/organisation.jsonnet

```diff
@@ -0,0 +1,571 @@
+(import "shared/default_format.jsonnet") + {
+  definitions: (import "shared/definitions/_whitehall.jsonnet") + {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        analytics_identifier: {
+          "$ref": "#/definitions/analytics_identifier",
+        },
+        body: {
+          "$ref": "#/definitions/body",
+        },
+        change_note: {
+          "$ref": "#/definitions/change_note",
+        },
+        brand: {
+          type: [
+            "string",
+            "null",
+          ],
+          description: "Deprecated.",
+        },
+        brand_colour: {
+          type: [
+            "string",
+            "null",
+          ],
+          pattern: "^#([0-9a-f]){6}$",
+          description: "The branding colour of the organisation, used for their logo and colour scheme.",
+        },
+        foi_exempt: {
+          type: "boolean",
+          description: "Whether the organisation is exempt from Freedom of Information requests.",
+        },
+        logo: {
+          type: "object",
+          properties: {
+            formatted_title: {
+              type: "string",
+            },
+            crest: {
+              type: [
+                "string",
+                "null",
+              ],
+              enum: [
+                "bis",
+                "dit",
+                "eo",
+                "hmrc",
+                "ho",
+                "mod",
+                "portcullis",
+                "single-identity",
+                "so",
+                "ukaea",
+                "wales",
+                null,
+              ],
+            },
+            image: {
+              "$ref": "#/definitions/image",
+            },
+          },
+          description: "The organisation's logo, including the logo image and formatted name.",
+        },
+        ordered_corporate_information_pages: {
```

Comment:
I think `ordered_corporate_information_pages` and `ordered_featured_links` are all migrated, could they go into the edition links?

Reply:
They are but they can also be external links so we need the flexibility...

Reply:
Ah that's unfortunate. We should make a note to represent the external links too ([see Mat's RFC](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-088-external-content.md)) 

Does this apply to `ordered_corporate_information_pages` as well?

Reply:
Yeah both `ordered_corporate_information_pages` and `ordered_featured_links` can contain external links - for example to the Civil Service Jobs site or organisation-specific services on other sites.